suite/tests/api/ir_x86.c

```diff
@@ -886,6 +886,31 @@ test_cti_prefixes(void *dc)
     disassemble_with_info(dc, buf, STDOUT, true, true);
 #endif
     ASSERT(decode_next_pc(dc, buf) == (byte *)&buf[6]);
+
+#if defined(DEBUG) && defined(BUILD_TESTS)
+    /* PR 4667, a c5 vex encoding where the second byte is c4
+     *   c5 c4 54 2d 68 1d 0c 0a vandps 0xa0c1d68(%rip),%ymm7,%ymm5
+     */
+    buf[0] = 0xc5;
+    buf[1] = 0xc4;
+    buf[2] = 0x54;
+    buf[3] = 0x2d;
+    buf[4] = 0x68;
+    buf[5] = 0x1d;
+    buf[6] = 0x0c;
+    buf[7] = 0x0a;
```

Comment:
OK.  I was thinking of a singe line that's a variant of one of these:
```
suite/tests/api/ir_x86_3args_avx.h:OPCODE(vandps_256, vandps, vandps, 0, REGARG(YMM0), REGARG(YMM1), MEMARG(OPSZ_32))
```
But I didn't look at what it takes to make it 0xc4: are there multiple encoding choices and DR doesn't ever pick the 0xc4 one?

In any case this should be sufficient.  Note that normally we would be point at the issue #4652 rather than the fixing PR.

Reply:
Ah I didn't try that.

Replaced the PR # with the Issue #.